korangar/src/main.rs

```diff
@@ -724,6 +724,10 @@ fn main() {
                         NetworkEvent::CharacterSlotSwitchFailed => {
                             interface.open_window(&application, &mut focus_state, &ErrorWindow::new("Failed to switch character slots".to_owned()));
                         },
+                        NetworkEvent::ResurrectPlayer(data) => {
+                            entities[0].set_idle(client_tick);
+                            interface.close_window_with_class(&mut focus_state, RespawnWindow::WINDOW_CLASS);
+                        }
```

Comment:
Seem like `data` here is actually the `player_id`.

Also, is this packet sent for other players too? If so, we should select the correct one from the list to resurrect and only close the window if the correct player (ourselves) was resurrected.

Reply:
There are more stuff to handle... There is the need to handle for other player dead and revival also.